### ISSUE TITLE
Downloading Song Issue

### ISSUE THREAD

**Issue by Eliya-Manoj** [HemantKArya/BloomeeTunes#56]
If i specify download folder, Downloading song does not work for me. 

Samsung M31
Android 12

**Comment by HemantKArya** [comment]
Yeah thanx @Eliya-Manoj for reminding about that issue, I am figuring about completely new package or library to solve this issue since there are no proper flutter package or solutions I got till now (you are welcome if come up with any sol.), which handles the permission for specific folder in android's new versions using (SAF).
So, till then this issue will be continued or let me know if anyone have solution idea for this.

**Comment by mkr-infinity** [comment]
![Screenshot_20241012-225213](https://github.com/user-attachments/assets/5d50c074-9619-434c-9ea7-26f6753edf80)

This for me now working any download

**Comment by HemantKArya** [comment]
> Yeah thanx @Eliya-Manoj for reminding about that issue, I am figuring about completely new package or library to solve this issue since there are no proper flutter package or solutions I got till now (you are welcome if come up with any sol.), which handles the permission for specific folder in android's new versions using (SAF).
> So, till then this issue will be continued or let me know if anyone have solution idea for this.

As I already explained but if you try to give bloomee explicitly all storage permission then it may solve this problem.
